unit-tests/src/test/java/org/eclipse/collections/impl/AbstractRichIterableTestCase.java

```diff
@@ -1042,7 +1051,7 @@ public void sumDoubleConsistentRounding2()
 
         Assert.assertEquals(
                 33333.0,
-                this.newWith(list.toArray(new Integer[]{})).sumOfDouble(i -> 1.0d / 3.0d),
+                this.newWith(list.toArray(new Integer[] {})).sumOfDouble(i -> 1.0d / 3.0d),
```

Comment:
Are these whitespace changes causing CheckStyle failures?

Reply:
Adding them didn't seem to cause any CheckStyles failures on the build. My Intellij CheckSyles plugin seemed to add these... I have just revert the addition.